openedx/features/content_type_gating/migrations/0009_auto_20210923_0935.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.24 on 2021-09-23 09:35
```

Comment:
Given that we need to edit the previous migrations anyway, it feels like we should be able to do it in a way that renders this migration unnecessary.  Just update the field definitions in the previous migrations to match the new verisons here, I don't think it should produce any actual database changes.